### DISCUSSION TITLE
Red Node Nuki Bridge

### DISCUSSION THREAD

**Discussion by Easykiwi** [community/community#35712] [Other Feature Feedback, Questions, & Ideas]
Hi all,
I try to create a Flow with node-red-contrib-nuki to integrate Nuki in my homematic ccu3, but Iˋm running from wall to wall 🙈

Have someone a working flow for me?

thx
Stefan

**Comment by Jemalabdi** [comment]
![Uploading 1660775151079.jpeg…]()
Very nice

**Comment by Easykiwi** [comment]
???
Look like something wrong with your attachment

**Comment by Easykiwi** [comment]
Habe nobody something for me 😢

**Comment by airtower-luna** [comment]
I'm afraid this might not be the best place to ask, the Community is mostly for discussions about Git, GitHub, and related topics. That doesn't mean other programming-related questions aren't allowed, but for something specialized you might have a better chance to get help in a place focusing on the topic you're asking about.

If the software you're trying to get help with is hosted on GitHub you should check its repository for support information (README or SUPPORT files, if they exist). If it doesn't list anything different, opening an issue or discussion *in that repository* is usually a good idea so the developers see it. :slightly_smiling_face:

**Reply by Easykiwi** [reply]
Thx I will do it so 😊

**Comment by ernestbuffington** [comment]
I think you need a hooberstank module for the muffler output. The flow should open up then!
